internal/core/thirdparty/tantivy/tantivy-binding/src/analyzer/tokenizers/tokenizer.rs

```diff
@@ -53,6 +53,18 @@ pub fn lindera_builder(
     Ok(TextAnalyzer::builder(tokenizer).dynamic())
 }
 
+pub fn char_group_builder(
+    params: Option<&json::Map<String, json::Value>>,
+) -> Result<TextAnalyzerBuilder> {
+    if params.is_none() {
+        return Err(TantivyBindingError::InvalidArgument(format!(
+            "char group tokenizer must be costum"
```

Comment:
`costum` is a typo.

change error to `char_group tokenizer must be customized.`